airflow/executors/kubernetes_executor.py

```diff
@@ -588,8 +644,31 @@ def sync(self) -> None:
             raise AirflowException(NOT_STARTED_MESSAGE)
         self.kube_scheduler.sync()
 
-        last_resource_version = None
```

Comment:
this resource_version is not needed here since it's not really used. The only place it's using it is the watcher process. But that's a different process. Info is not communicated. Therefore, it's safe to delete